apps/nextjs/package.json

```diff
@@ -7,9 +7,10 @@
     "build:dev": "pnpm with-env next build",
     "check": "tsc --noEmit",
     "clean": "rm -rf .next .turbo node_modules",
-    "dev": "concurrently \"pnpm dev-server\" \"node scripts/local-dev.mjs\"",
-    "dev-turbo": "pnpm with-env node scripts/increase-listeners.js next dev --port 2525 --turbo | pino-pretty -C",
-    "dev-server": "pnpm with-env node scripts/increase-listeners.js next dev --port 2525 | pino-pretty -C",
+    "dev": "FORCE_COLOR=1 concurrently \"pnpm dev-server\" \"node scripts/local-dev.mjs\"",
+    "dev-turbo": "FORCE_COLOR=1 concurrently \"pnpm dev-server-turbo\" \"node scripts/local-dev.mjs\"",
```

Comment:
For my benefit, what tool needs FORCE_COLOR? Is it because concurrently is getting in the way? Could you add it to `.env` like `DEBUG_COLORS`?

I'm not convinced that we actually need the concurrent local dev server any more. I haven't noticed any issues having it disabled in the last week